src/functions/Output.ps1

```diff
@@ -57,6 +58,10 @@ $script:ReportTheme = DATA {
         Foreground       = 'White'
         Information      = 'DarkGray'
         Coverage         = 'White'
+        CoverageWarn     = 'DarkRed'
```

Comment:
This does not seem to be used. If it is used and I just don't see it, warning should be yellow probably.

Reply:
@asears ping on this.

Reply:
Makes sense, removed as not currently in use.